src/low-preview/ci.service.js

```diff
@@ -154,6 +154,33 @@ export default class CIResponsive {
       { isUpdate, imgNode, ratio, lazy, placeholderBackground, preserveSize, isPreview, ...imgProps }
     );
 
+    const observer = new MutationObserver(mutations => {
```

Comment:
we should implement a new function as mentioned in the card setSrc this function takes two arguments first the element which we need to set the src and the src value